video_store.py

```diff
@@ -0,0 +1,156 @@
+import requests 
+import datetime 
+
+class Videostore:
+    def __init__(self, url="http://localhost:5000", selected_video=None, selected_customer=None):
+        self.url = url
+        self.selected_video = selected_video
+        self.selected_customer = selected_customer 
+
+# ==========All For Video Options========== 
+
+    # Option 1: Add a video 
+    def create_video(self,title="default", release_date="default", total_inventory=None):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
+            "total_inventory": total_inventory
+        }
+        response = requests.post(self.url+"/videos",json=query_params)
+        return response.json()
+
+    # Option 2: Edit a video 
+    def update_video(self,video_id="default", title="title", release_date="default", total_inventory=None):
+        if not title:
+            title = self.selected_video["title"]
+        if not release_date:
+            release_date = self.selected_video["release_date"]
+        if not total_inventory:
+            total_inventory = self.selected_video["total_inventory"]
```

Comment:
Cool that you have optional support for using the already selected video's data for fields we don't care to update!